docs/topics/i18n/translation.txt

```diff
@@ -2196,3 +2196,21 @@ to produce the binary ``.mo`` files that are used by ``gettext``.
 You can also run :djadmin:`django-admin compilemessages
 --settings=path.to.settings <compilemessages>` to make the compiler process all
 the directories in your :setting:`LOCALE_PATHS` setting.
+
+Non-English base language
+-------------------------
+
+Django makes the general assumption that the original strings in a translatable
+project are written in English.
+You can choose another language, but you must be aware of certain limitations:
+
+* ``gettext`` only provide two plural forms for the original messages, so you
+  will need to also provide a translation for the base language to include all
+  plural forms if the plural rules for the base language are different from
+  English.
+
+* When an English variant is activated and English strings are missing, the
+  fallback language will not be the :setting:`LANGUAGE_CODE` of the project,
+  but the original strings. For example, an English user visiting a site which
```

Comment:
visiting a site with Spanish as the default language and original strings written in Russian will...